### DISCUSSION TITLE
2023-11-24 - Less is more - Visual - Chewing the Fat 📈

### DISCUSSION THREAD

**Discussion by bradystroud** [SSWConsulting/SSW.Rules.Content#7303] [Chewing the Fat]
Hey all 🎨,

Lets discuss this rule
https://www.ssw.com.au/rules/less-is-more/

Where can we implement this at SSW?

**Comment by piers-sinclair** [comment]
I think that our posters often have too much info on them, I would like to see more concise posters with limited text using visuals to communicate as much as possible.

**Reply by tiagov8** [reply]
Hey @pierssinclairssw , 

1. Please add some links/screenshots to illustrate ;)

**Reply by Geordie88** [reply]
Overloading visual products with text is a problem (see user group thumbnails and adams blog images). I have noticed a positive trend in this area but we still have a ways to go.

**Comment by jernejk** [comment]
On TimePro, having fewer fields would be nice, especially for suggested timesheets, where notes are usually the only thing that needs to be changed.

![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/5943653/3bbb7014-721e-418f-bc99-d6b633d68821)
**Figure: Click on edit timesheet, there should be a quick edit button.**

![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/5943653/b632e078-c3c4-42ad-8d2c-3c7a61612bfc)
**Figure: Get a popup to quickly edit commonly changed data on timesheets.**

**Reply by christoment** [reply]
Also adding to edit timesheet page - we have information on this page that people rarely touch or even change (e.g. billing section).
Would be nice if we can revisit this page and reduce the clutter 😊

**Reply by SamBWagner** [reply]
I second this and have made a PBI for moving/removing the oodles of dead space and unused/barely used features on the TimePro home page
https://dev.azure.com/ssw2/SSW.TimeProDotNet/_workitems/edit/53162

**Comment by tiagov8** [comment]
### SSW Events

**Less (sites) is more (consistency)**

I don't think we should have extra HackDays websites (e.g. https://aihackday.com/, https://angularhackday.com/)

We should have all events under SSW Events (https://www.ssw.com.au/events). 
Easier to keep consistency and most importantly easier to maintain.

**Note:** The same is true for https://sswdigital.com/

**Reply by GordonBeeming** [reply]
Also, having that content under the ssw.com.au domain bumps up the search juice 😁... or lowers it if we doing dodgy things, but likely bumps it up

**Reply by wicksipedia** [reply]
We can still keep the hackday urls and direct them to the event pages

**Reply by sethdaily** [reply]
+1

**Comment by tiagov8** [comment]
### SSW Contact Form

We should avoid repetition of the word "Your"
I also think the question "How did you hear about us" to not be very valueable,  but I might be wrong

Issue: https://github.com/SSWConsulting/SSW.Website/issues/763

<img width="1029" alt="Screenshot 2023-11-23 at 1 42 49 PM" src="https://github.com/SSWConsulting/SSW.Rules.Content/assets/12601228/89e2a543-2b00-46d1-b5ab-b45c3e99e0d0">

**Comment by bahjatalaadelssw** [comment]
This is a great method to follow in order to keep products clear and on message.

**Comment by ozdemir-mehmet** [comment]
If less is more, can we make shorts and t-shirts a thing every day? But the best change I think was the stage, followed by the thumbnail.

**Comment by AlexRothwell** [comment]
The total number of websites that SSW has for various reasons should have this principle applied so we can focus on maintaining a single, modern and high quality site. What is currently done through multiple separate websites could be done with different landing pages on the core site(s) instead

**Comment by jeoffreyfischer** [comment]
We should apply this principle as often as possible, because it helps keeping the user or the audience's attention for longer.

**Comment by jakebayliss** [comment]
Great rule and one I have been trying to follow for a while. Can be applied to almost anything, not just design

**Comment by Levijj22** [comment]
Generally speaking, I see the importance of "less is more". However I wasn't a fan of the final thumbnail result. Although it was originally overcrowded, I feel like too much is missing now. Although the after result looks tidier, I worry that the main message is now missed. 
For e.g. the old thumbnail was speaking to Developers specifically, now it's too general. Also, there was a nice call-to-action with the .NETUG logo, however now the reader has to spend more effort to work out how to "Grow their career", and then they work out its only for developers. 

That being said, nice video Joseph! Less is

**Reply by Levijj22** [reply]
p.s. where did you get that jacket?? 🧐

**Comment by JeanThirion** [comment]
Time to review some old projects I guess... Brilliant video Joseph !

**Comment by Marxoz** [comment]
"Less is more" is such a good mantra to have, but I think it has its place. Take a website as example. You don't want it to be too crowded, but at the same time you want to display as much info as possible to minimise clicks, and to show the user what they are looking for in a single page.

The thumbnail is another example. I don't know if it is a user group, a ted talk, or a document that I will be downloading when I see it. I know there will probably be a description with the thumbnail, but the visual will catch my attention before the text does.

Also, good to see Brady almost smiling

**Comment by joshbermanssw** [comment]
Great design principle - small things that make tasks more enjoyable. With examples like the thumbnail, abiding by the principle also makes it look more professional than the pre-edited version.

**Comment by matt-goldman** [comment]
The main one for me is the stage. Joseph already showed a concept for a less-cluttered look, which will help us keep focus on the presenter. I'd like to take it a step further and get rid of the desk.

And leading on from here - I think we can go "less is more" in our presentations too. For an example, see my recent talk on my book, which will hopefully be on the channel soon.

I love this design principle. Having worked in health for 10 years and seeing the cluttered UIs in all the software there, I feel this in my bones. I think at SSW we're quite good at this, but there's always room to improve.

**Reply by leahy268** [reply]
I agree the stage was the best change.  Though I disagree about removing the desk.  I would want that optional at least..  For a lot of people there is some comfort by being behind a desk..

**Comment by tkapa** [comment]
I'd love to see that stage design come to life, I don't think the video could do it proper justice.

Bad UI will lead to bad UX. 

I try to prioritise usability when I can.

**Comment by TobyChurches** [comment]
A great rule that was really well presented!

Clean UI is something that is often overlooked. It can be tricky finding that balance between displaying all the critical information without cluttering the page. I find that simple UI generally lead to a better UX, with overly cluttered pages often loosing a sense of legitimacy (sometimes feel like a scam)

**Comment by Ben0189** [comment]
Nice video from Joseph, i like how to stage is changed after applied less is more

**Comment by MarcusIrmscherSSW** [comment]
Great Video Joseph. I think in most cases the principle of less is more is a good thing. But every rule has its exceptions. :)

**Comment by tombui99** [comment]
An awesome video from Joseph! 
The new design and layout for the stage looks significantly better, the SSW logo actually stands out a lot more compared to having it all over the place. With regards to the Less is More concept, we definitely should have a major UI redesign for all current SSW projects to focus on reducing clutters.

**Comment by AttackOnMorty** [comment]
Great video and rule. It would be helpful to have some more practical rules to follow when applying "Less is More".

**Comment by JackDevAU** [comment]
"Less is more" is a great principle and we at SSW should apply it more!

**Comment by samsmithssw** [comment]
Great vid Joseph!!!

I think the China website is a great example of this principle, the home page is a lot less cluttered and busy than ssw.com.au once you scroll down

<img width="2032" alt="Screenshot 2023-11-24 at 09 43 25" src="https://github.com/SSWConsulting/SSW.Rules.Content/assets/115435510/34dcb43c-8f22-4edd-ae07-76fb1fff306d">

**Figure: ❌ ssw.com.au is cluttered when you scroll down, a lot of walls of text**

<img width="2032" alt="Screenshot 2023-11-24 at 09 43 20" src="https://github.com/SSWConsulting/SSW.Rules.Content/assets/115435510/908770c4-53e7-4ce7-b5fb-a327467709d4">

**Figure: ✅ ssw.cn is vissually appealing, essential info only and easy to read (even though it's in Chinese!)**

**Comment by Calinator444** [comment]
I had a difficult time thinking of specific examples where the Less is more principle is poorly applied in any of SSW's internal projects. I do have some feedback for the SSW rules edit screen though. Please let me know what you think.


<img width="1097" alt="SSW rules feedback" src="https://github.com/SSWConsulting/SSW.Rules.Content/assets/65635198/eceb6176-c3cc-45c0-ba9e-3a0c25517537">

**Figure**: Suggestions for improvements for SSW rules

**Comment by UlyssesMaclaren** [comment]
TimePro timesheets. 

1. Change from 
    time start, time end, time less, time total
to
    time total

This would simplify data entry, reporting, invoicing, error handling, calculations, etc.

FYI these times do not show up on invoice PDFs and no client has ever complained... so we know this is not required data for customers.

**Comment by AshAnil** [comment]
Great video. The thumbnail design with more visual elements is better and more clear what the video is all about.

**Comment by sethdaily** [comment]
SSW Website megamenu, Consulting index, video thumbnails, User Group social media designs, SSW Events websites (should have no hackday sites)

**Comment by ricksu978** [comment]
### SSW News & Press
https://www.ssw.com.au/company/news
On this page, we have included ALL the news back in 1990, I think it is a good place to apply "less is more" principle

![www ssw com au_company_news](https://github.com/SSWConsulting/SSW.Rules.Content/assets/35659961/8c990faa-0bd0-4b94-94d3-d3310ba02d49)
Figure: too many news in one page.

**Comment by leehawkinsssw** [comment]
The videos page on the SSW TV YouTube channel is an example of where cleaner thumbnails would really make a big difference, as it's visually overwhelming (and very inconsistent) currently:

![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/106290015/008e28f3-a84c-452e-bbda-16e7694c5b20)
**Figure: lots of noisy thumbnails on the SSW TV YouTube channel "Videos" page**

**Comment by vladislav-kir** [comment]
Less is More is an amazing rule, and I loved the video from Joseph. All the examples were really nice :)

I think TimePro Edit timesheet page could simplified, especially Billing Section. 
I know it can be collapsed now, but I think the read-only fields there, like ADMIN NOTES, could be hidden altogether for majority of the users.

**Comment by SamBWagner** [comment]
Amazing video, short, sharp and to the point!

**Comment by tomek-i** [comment]
I'm a big fan of minimalistic designs.

What also helps with video is to have a wide aperture to blur the backgrounds out or do it in post. The example image of the office it was better but the chairs where still visible, people in the background etc. in that case i would have blurred the background out to draw focus back to the center / speaker etc.

I think on the home page, keep the TOP section, then remove the spill about <3 microsoft and put it on an ABOUT page and put the upcoming events there in a large carousel. alternatively do the same and move the block of texts under the carousel of event. 
I
![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/3699937/2933e7a0-cbed-4ea4-a3e9-3ee0e412ac61)

**Comment by YalalCakirSSW** [comment]
Great video and great rule. We should use this rule in our daily life as well.

**Comment by JackDuan123** [comment]
SSW rules

**Comment by ncn-ssw** [comment]
The video was fantastic. I do have a concern about the video thumbnail though - the minimalist result does not have the distinctive SSW feel, and that feel is an important part of our marketing.

**Comment by cvionnet** [comment]
Great video and rule Joseph! 👍
And impressive work with the office background transformation.

**Comment by synnoveisabel** [comment]
Great video and a great principle to keep in mind in everything from design to daily life. The stage look great with a cleaner design 🚀 
I think less links in READMEs would make it easier to follow the content.

**Comment by steven0x51** [comment]
Great rule! I love this mantra and I believe that "simplicity is the ultimate sophistication". But including this rule to our rules, there is a conflict

![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/41563445/224db80a-e1c8-473a-9eb8-0d46ef0c78fd)
**Figure: More is less?**

![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/41563445/cf826b07-0b11-4c5c-ab16-01e3e0f74d3f)
**Figure: Example of some irrelevant rules**

**Comment by amankumarrr** [comment]
Joseph, your video was fantastic! I've always been a huge admirer of well-designed solutions with robust functionality. It seems like this principle applies almost everywhere.

**Comment by bettybondoc** [comment]
The TimePro interface appears cluttered on my end. Numerous unnecessary fields demand attention despite not needing to be filled out. Consequently, I find myself scrolling extensively before being able to add a Timesheet.

![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/33026270/652c592d-bd4d-4420-955f-1c0b4fde8044)
**Figure: Where I land whenever I add a new timesheet**

![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/33026270/56df3cd1-b7d8-4d12-b7af-58c7f975b7cf)
**Figure: Unnecessary fields that could be collapsed so I can get to the "Add" button quicker**

**Reply by tomek-i** [reply]
Bettttyyyyy :-) we need to find out why you have the old layout, when i started with SSW thats the suggestion i made and they made it collapsible to hide it away 

![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/3699937/3f85252f-90b7-4da6-9cbc-cf96da7524a3)

**Reply by bettybondoc** [reply]
That looks much better!

**Comment by Dhruv-0987** [comment]
Website just bellow the banner: 
<img width="1498" alt="image" src="https://github.com/SSWConsulting/SSW.Rules.Content/assets/69164713/c534feb7-a6f8-46b2-a650-5c7c0a01d589">
Figure: SSW Website Home page - tiles below the banner.

No need to write the text below the title and have better icons. Anyone who comes our website have some preliminary knowledge of what each of those heading entail.

**Comment by MattParkerDev** [comment]
I agree that the stage redesign would be good, and the thumbnail design is very nice

**Comment by Harry-Ross** [comment]
I think this is a super important rule to keep in mind when designing UI, and a great video! 

I think it would be great to combine a lot of the existing SSW Websites (Rules, People, Hack Days, etc) into the main SSW.Website repo, either by migrating these to Next.js or by implementing a monorepo (for easier component reuse between projects).

**Comment by PennyWalker** [comment]
I'm a big fan of less is more. I think we regularly try to fit too much into one location - our Chat GPT poster is a classic example where people won't digest the info because there is too much noise. I think it's crazy to start any development project without using a designer to help - often UI/UX issues could have been avoided if they'd been brought in from the beginning.

**Comment by CoffeeBugHub** [comment]
Great video, concise and appealing. 
And the examples showing in the video are very persuasive as well.

**Comment by jaydenalchin** [comment]
I think SSW leans on a more-is-more approach quite often which can lead to painful UX (see how many Rules, SugarLearning items, and web pages we create & maintain).

I definitely feel we're getting better and moving in the right direction, but there's still much room for improvement.

Our posters & graphics could be simpler, cleaner & punchier. A lot of our web pages could as well:

https://github.com/SSWConsulting/SSW.Rules.Content/assets/75593567/c5f243b2-80df-4b1b-baae-072efee3c949

**Video: Bad example - SSW People page**

<img width="1172" alt="image" src="https://github.com/SSWConsulting/SSW.Rules.Content/assets/75593567/f72893f8-cb01-4dbf-b51a-9e63e1caf594">

**Figure: Bad example - Megamenu can still be simplified down**

<img width="1403" alt="image" src="https://github.com/SSWConsulting/SSW.Rules.Content/assets/75593567/a730b5d5-02d5-4760-a059-aa9ab00f3770">

**Figure: Bad example - Crowded homepage**

**Comment by ozair-ashfaque** [comment]
Great video and rule Joseph

**Comment by jimmy1992abc** [comment]
If we have an employee portal that manages all of our SSW stuff (Aka gamification SSW TimePro plus plus, where you can manage yourself like a game character online ), We could remove more than 50% of SugarLearnings or SSW Rules.

**Comment by chrisschultzssw** [comment]
https://www.ssw.com.au/rules/ has all the categories there on the "home" page. I'm not sure what the solution is - but there is probably a way to show less, while making it easier to find the rule/category you're after.

**Reply by tiagov8** [reply]
New Rules site is coming and @jaydenalchin has found a great solution for this ;)

<img width="593" alt="Screenshot 2023-11-24 at 2 30 55 PM" src="https://github.com/SSWConsulting/SSW.Rules.Content/assets/12601228/8f964921-d0a2-4400-8037-893a9bf536a4">

**Comment by Aibono1225** [comment]
love this video and the concept - fewer distractions, more clarity

**Comment by KristenHu** [comment]
"Less is more" is a great concept, but sometimes clients still want to showcase too much information, in which situation, this rule would unavoidably be broken.

**Comment by sylhuang** [comment]
I agree with the other posts that we have a bit too many rules. Harder to read them all and remember the most important ones..

**Comment by hveraus** [comment]
One lesson I learned from Less is More is Done design is achieved not when there is nothing more to add, but when there is nothing left to take away.

**Comment by AnkyChe** [comment]
Love the examples shown in the video, too much information on the same page just makes it hard to read. I think it also applies to Excel sheets.

**Comment by babakamyljanovssw** [comment]
Great video and rule! Agree with "Thumbnail" design, it made much easier to understand the context

**Comment by tino-liu** [comment]
I agree with "Less is More". But even if it is less, the title needs to be captivating with enough context.

**Comment by zl24622** [comment]
"Less is More" is not only for designing but also applied to products (less features) and coding (keeping solutions simple).

**Comment by Freego1783** [comment]
Great video and rule ! Thanks for this one Joseph.

Finding balance in UI is not an easy task but that's clearly somehting we should all work on.

**Comment by BrookJeynes** [comment]
less is more not only relates to designs but can relate nicely with programming as well

**Comment by danielmackay** [comment]
Such a simple concept that can have a dramatic effect in many areas of software development (not just UI).

**Comment by stephanfakossw** [comment]
I love this concept, when it's balanced :-)

**Comment by jackreimers** [comment]
Great rule and very important to follow to ensure users/viewers are not overloaded and can understand the function or message of a piece of content.

**Comment by PothieuG** [comment]
As a strong defender of "less is more" concept, I loved everything I saw on that Chewing the fat !
